lib/engine/game/g_18_eus/step/dividend.rb

```diff
@@ -11,7 +11,26 @@ class Dividend < Engine::Step::Dividend
           DIVIDEND_TYPES = %i[payout half withhold].freeze
           include Engine::Step::HalfPay
 
+          ACTIONS = ['dividend'].freeze
+          def actions(entity)
+            return super unless entity == @game.bny
+
+            ACTIONS
```

Comment:
Is this needed for the auto action to work?

Reply:
Looks like it is not